app/views/stories/tagged_articles/index.html.erb

```diff
@@ -130,5 +138,5 @@
     <%= render "stories/tagged_articles/sidebar_additional" %>
   </div>
 
-  <%= javascript_packs_with_chunks_tag "storiesList", "followButtons", "feedPreviewCards", "hideBookmarkButtons", "drawerSliders", "localizeArticleDates", defer: true %>
+  <%= javascript_packs_with_chunks_tag "storiesList", "followButtons", "tagFollows", "feedPreviewCards", "hideBookmarkButtons", "drawerSliders", "localizeArticleDates", defer: true %>
```

Comment:
Some of the changes in this PR is to ensure that we can use tagFollows and followButtons in conjunction. I did some renaming of components as well. As discussed previously, I'm hoping that we can at some point make the TagButton component a general component and use it in place of FollowButtons because Preact does so much of the heavy lifting for us that followButtons is doing manually. But for now I didn't try to name these files too generally because I wanted to avoid confusion whilst we're in this intermediate phase.